From 5df8b2a18703fb16bd72f1a5ec247ad3a3dbbbab Mon Sep 17 00:00:00 2001
From: Kevin McCarthy <kevin@8t8.us>
Date: Mon, 6 Apr 2020 14:53:42 -0700
Subject: [PATCH 05/19] Turn on READONLY flag for postpone count check.

We don't manipulate the context, so I don't think this is fixing an
actual bug.  But better to be explicit about it.
---
 postpone.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/postpone.c b/postpone.c
index 3cbc4d03..f557976d 100644
--- a/postpone.c
+++ b/postpone.c
@@ -134,7 +134,8 @@ int mutt_num_postponed (int force)
 
     if (access (Postponed, R_OK | F_OK) != 0)
       return (PostCount = 0);
-    if (mx_open_mailbox (Postponed, MUTT_NOSORT | MUTT_QUIET, &ctx) == NULL)
+    if (mx_open_mailbox (Postponed, MUTT_NOSORT | MUTT_QUIET | MUTT_READONLY,
+                         &ctx) == NULL)
       PostCount = 0;
     else
       PostCount = ctx.msgcount;
